02_bag.ipynb

```diff
@@ -623,9 +623,9 @@
  "metadata": {
   "anaconda-cloud": {},
   "kernelspec": {
-   "display_name": "Python [conda env:dask-dev] *",
+   "display_name": "Python 3",
    "language": "python",
-   "name": "conda-env-dask-dev-py"
+   "name": "python3"
```

Comment:
i don't think these changes matter as they will get loaded again when the session starts, correct ?  wanted to double check

Reply:
That's correct.  Do you want me to remove them?

Reply:
No, I think it's fine.  wanted the confirmation